docs/jumpstart/week-7.md

```diff
@@ -0,0 +1,512 @@
+export const clapPost = async (id: number): Promise<void> => AxiosInstance.post(`${POSTS}/${id}/clap`);
+```
+
+Just like before we have an anonymous function that has an `AxiosInstance` to make the request. The
+difference is that this time the function takes in an argument. This argument is going to be the
+data that we want to post to the backend. Now, to make a post request, `post()` takes in the url 
+location of where to post the data, and the data to post at that location in the form 
+`post(location,data)`. Let's look specifically at `createPost`. We know that we want to "post" the 
+blog post to the backend. So, we take in a post as an argument. In the `post()` call we give the 
+url (which in this case is `"/posts"`), and the data we want to post (the argument we took in the 
+anonymous function). 
+
+That's all there is to it.   
+
+##### Delete Request
+
+The delete request will be the last type of request that we will use. What do you think we can use
+it for? Right again! We can use the delete request to delete a post/comment. Let's take a look at
+how to do that:
+
+```
+export const deletePost = async (id: number) => AxiosInstance.delete(`${POSTS}/${id}`);
+``` 
+
+Same as before. We have an anonymous function that takes in the post number to delete. We then use
+our `AxiosInstance` to call delete the id from the specific url. It is simliar to our get request
+excpet we are not getting but deleting.
+
+### Default Exports vs Named Exports
+
+You may have noticed that we export each axios call separately by name instead of just using
+`export default` at the end of the file. Exporting an individual item is called a named export while
+exporting a whole file is called a default export. The difference between the two is how we import
+them into our other files. A named export has to be imported with curly braces like so:
+
+```
+import { namedExport } from 'things';
+```
+
+Our default exports do not need the curly braces:
+
+```
+import defaultExport from 'otherThing';
+```
+
+> What imports that we use currently are named exports? What about default exports?
+
+
+### Calling Requests in our application
+
+Using our request is very simple. Let's see how to use our get our `getAllPosts`.
+
+```
+getAllPosts().then((p: Post[])=>{
+    setPosts(p);
+})
+```
+
+We first start by importing request we want from the api file. For the `getAllPosts` request our
+import would look like this:
+
+```
+import { getAllPosts } from '../api/api';
+```
+
+>We have the curly braces because it is a named export
+
+Next we just use the import like a function (because it is one!). All of our request that we made
+in the `api.tsx` file are just axios promises. We now need to further resolve those promises using
+`.then()`. Here we are simply use `.then()` to set the posts state to the posts that we get back
+from the request!
+
+It is as simple as that.
+
+## JEST
```

Comment:
Delete this? Or you can throw in a little sentence if you want to introduce it real quick